test/unit/specs/store/media-store.spec.js

```diff
@@ -53,6 +57,7 @@ describe('Search Store', () => {
 
     beforeEach(() => {
       state = store.state()
+      setActivePinia(createPinia())
```

Comment:
I'm unclear of when it's necessary to use `setActivePinia`. Could you add a blurb in the TESTING_GUIDELINES explaining that (or else open an issue to do it in a separate PR)?